composer.json

```diff
@@ -53,7 +53,7 @@
         "cs-fix": "phpcbf",
         "stan": "vendor/bin/phpstan analyze",
         "stan-baseline": "vendor/bin/phpstan analyze --generate-baseline",
-        "psalm": "vendor/bin/psalm.phar --show-info=false",
+        "psalm": "vendor/bin/psalm.phar --config=psalm.xml",
```

Comment:
Do we need the config? Isnt it defaulting to this file?

Reply:
We probably don't need it, but there are multiple configs and it wasn't clear if there were issues with which config was being targeted while debugging.  I didn't see any harm in leaving it.